mu-plugins/blocks/language-suggest/src/block.json

```
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "wporg/language-suggest",
+	"version": "0.1.0",
+	"title": "Language Suggest",
+	"category": "widgets",
+	"icon": "admin-site-alt3",
+	"description": "A block for use across the whole wp.org network.",
+	"textdomain": "language-suggest",
+	"editorScript": "file:./index.js",
+	"viewScript": "file:./front.js",
+	"style": "file:./style.css",
+	"supports": {
+		"align": true,
+		"color": {
+			"background": true,
+			"text": true,
+			"link": true
+		},
+		"typography": {
+			"fontSize": true
+		}
```

If we don't expect to have more than one, and the JS in `front.js` doesn't work with more than one, we can enforce that with a block supports setting

```suggestion
		},
		multiple: false
```